src/seraphis_core/jamtis_enote_utils.cpp

```diff
@@ -253,12 +307,12 @@ void make_jamtis_sender_receiver_secret_plain(const crypto::x25519_secret_key &p
 void make_jamtis_sender_receiver_secret_selfsend(const crypto::secret_key &k_view_balance,
     const crypto::x25519_pubkey &enote_ephemeral_pubkey,
     const rct::key &input_context,
-    const JamtisSelfSendType self_send_type,
+    const bool is_auxiliary_self_send_type,
     rct::key &sender_receiver_secret_out)
 {
-    // q = H_32[k_vb](xK_e, input_context)
-    SpKDFTranscript transcript{selfsend_sender_receiver_secret_domain_separator(self_send_type), 2*sizeof(rct::key)};
-    transcript.append("xK_e", enote_ephemeral_pubkey);
+    // q = H_32[k_vb](is_aux, D_e, input_context)
```

Comment:
```suggestion
    // q = H_32[k_vb](D_e, input_context)
```
`is_aux` isn't hashed, it's implicit in the domain separator.